features/step_definitions/individual_steps.rb

```diff
@@ -1023,6 +1023,11 @@ class VlpDocument
   expect(page).to have_content "My #{Settings.site.short_name}"
 end
 
+Then(/Individual should land on help paying coverage page$/) do
+  sleep 1
+  expect(page).to have_content "Upload document(s) proving your identity"
```

Comment:
please use a page object reference or a css class/attribute, but not hard-coded english text